File: Code 3 Callouts/Code 3 Callouts/Models/Peds/Victim.vb
Imports Rage

Namespace Models.Peds

    Public Class Victim
        Inherits PedBase

        Public Sub New(ByVal pName As String, ByVal position As Rage.Vector3)
            MyBase.New(pName, PedType.Victim, position)
        End Sub

        Public Sub New(ByVal pName As String, ByVal model As Rage.Model, ByVal position As Rage.Vector3, ByVal heading As Single)
            MyBase.New(pName, PedType.Victim, model, position, heading)
        End Sub

        Protected Friend Sub New(ByVal pName As String, ByVal handle As Rage.PoolHandle)
            MyBase.New(pName, PedType.Victim, handle)
        End Sub

    End Class

End Namespace

File: Code 3 Callouts/Code 3 Callouts/Models/Peds/Victim.cs
using Rage;
using static Stealth.Plugins.Code3Callouts.Common;

namespace Stealth.Plugins.Code3Callouts.Models.Peds
{

	internal class Victim : PedBase, IPedBase
	{

		internal Victim(string pName, Rage.Vector3 position) : base(pName, PedType.Victim, position)
        {
			
		}

		internal Victim(string pName, Rage.Model model, Rage.Vector3 position, float heading) : base(pName, PedType.Victim, model, position, heading)
        {
			
		}

		protected internal Victim(string pName, Rage.PoolHandle handle) : base(pName, PedType.Victim, handle)
        {
			
		}

	}

}